client/src/main/com/sinch/sdk/domains/conversation/api/v1/MessagesService.java

```diff
@@ -49,22 +49,85 @@ public interface MessagesService {
    */
   SendMessageResponse sendMessage(SendMessageRequest<? extends AppMessageBody> request);
 
+  /**
+   * Send a card message
+   *
+   * @param request CardMessage request
+   * @return Response
```

Comment:
This javadoc is strange:
 * for the parameter, the word `request` is repeated and `CardMessage` is not really the class to use
 * for the return, the response is a `SendMessageResponse` instead of a `Response`

Reply:
> This javadoc is strange:
>     * for the parameter, the word `request` is repeated and `CardMessage` is not really the class to use

Yes, here the 1st 'request' is the parameter name from function prototype. Then `CardMessage request` is the message used for documentation
javadoc output:
![image](https://github.com/user-attachments/assets/0dcd6d8c-74a9-49b7-962e-7287e8c4035b)

> for the return, the response is a SendMessageResponse instead of a Response

Purpose here was not document return type (javadoc do it automatically from prototype)
Changed in favor of a true sentence
![image](https://github.com/user-attachments/assets/72476395-8a7b-42a4-a2f2-3f9cdf5f11b1)